anesthetic/samples.py

```diff
@@ -62,57 +60,56 @@ class MCMCSamples(WeightedDataFrame):
         Anything equal or below this value is set to `-np.inf`.
         default: -1e30
 
-    burn_in: int or float
-        Discards the first integer number of nsamples if int
-        or the first fraction of nsamples if float.
-        Only works if `root` provided and if chains are GetDist compatible.
-        default: False
-
     """
 
+    _metadata = WeightedDataFrame._metadata + ['tex', 'limits', 'root',
+                                               'label']
```

Comment:
Should root be here if we're removing the reading functionality? This may likely change if we implement #208

Reply:
Agreed, updated in [02aae9e](https://github.com/williamjameshandley/anesthetic/pull/209/commits/02aae9edc34c2a861cc250c66dcf77e17821ccf5).